.github/workflows/test.yml

```diff
@@ -378,7 +379,6 @@ jobs:
 
   e2e:
     name: e2e
-    if: github.event_name != 'schedule'
```

Comment:
so basically the cron schedule above that makes it run every 12 hours each day is going to kick in?

Reply:
yes.